products/sle/main.pm

```diff
@@ -572,6 +572,7 @@ sub load_consoletests() {
         loadtest "console/vim.pm";
         if (!is_staging()) {
             loadtest "console/firewall_enabled.pm";
+            loadtest "console/zypper_lifecycle.pm";
```

Comment:
software selection has an impact on this test, as additional addons should produce an output for those products

This is especially important for modules, which have a different lifecycle from the rest of the product (the whole point of this feature)